core/src/main/java/org/commonjava/indy/core/ctl/PaginationHandler.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2011-2017 Red Hat, Inc. (https://github.com/Commonjava/indy)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.core.ctl;
+
+import java.util.List;
+
+/**
+ * Created by yma on 2018/1/15.
+ */
+public interface PaginationHandler<T>
+{
+    T getCurrData( int pageIndex, int pageSize );
+
+    List<T> getCurrDataList( int pageIndex, int pageSize );
```

Comment:
What is this used for?

Reply:
I keep this method for further list extension, but I'm aware it should be redundant, handler instances could cover that using the generic paradigm and do its own loop business, I'll remove this in follwing commit.